airbyte-integrations/connectors/source-amazon-seller-partner/integration_tests/configured_catalog_browse_tree_data.json

```diff
@@ -0,0 +1,106 @@
+{
+  "streams": [
+    {
+      "stream": {
+        "name": "GET_XML_BROWSE_TREE_DATA",
+        "json_schema": {
+          "title": "XML Browse Tree Data",
+          "type": "object",
+          "$schema": "http://json-schema.org/draft-07/schema#",
+          "properties": {
+            "browseNodeId": {
+              "type": ["string"]
+            },
+            "browseNodeAttributes": {
+              "type": ["null", "object"],
+              "properties": {
+                "attribute": {
+                  "type": ["array"],
+                  "items": {
+                    "type": ["null", "object"]
+                  },
+                  "properties": {
+                    "name": {
+                      "type": ["null", "string"]
+                    },
+                    "text": {
+                      "type": ["null", "string"]
+                    }
+                  }
+                },
+                "count": {
```

Comment:
@manescianera I saw you renamed to `count` but I'm not sure you changed the parsing logic to rename the `@count` to `count` in the XML output. Did I miss something?

Reply:
`attr_prefix` argument takes care of the json prefixes from parsed XML node properties, so I've changed `attr_prefix` to a blank string over [here](https://github.com/airbytehq/airbyte/blob/754c2bbb632e56b3d3a2e5d11dfb80ce8bb9da57/airbyte-integrations/connectors/source-amazon-seller-partner/source_amazon_seller_partner/streams.py#L409)